primer/primer.cabal

```diff
@@ -79,6 +79,7 @@ library
     Primer.Eval.Let
     Primer.Eval.NormalOrder
     Primer.Eval.Prim
+    Primer.Eval.Push
```

Comment:
A note on the commit message of 'feat: evaluate lets by "pushing down"':  there is a note here about "document the future simplifications" -- I'll reword this later once I have actually done some such